java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -1451,6 +1451,7 @@ else if ("debian".equalsIgnoreCase(grains.getValueAsString("os"))) {
             server.setOsFamily(grains.getValueAsString("os_family"));
             server.setRunningKernel(grains.getValueAsString("kernelrelease"));
             server.setOs(grains.getValueAsString("osfullname"));
+            server.setCpe(grains.getValueAsString("cpe"));
```

Comment:
Asking for feedback: New systems will be assigned a CPE as part of the registration process, however, old systems must be migrated. The idea here is to set the CPE when users refresh their package list. The package refresh process can be slow, and users have to refresh the package list at least once for each of their systems to use the CVE auditing functionality. This is especially a concern for users with hundreds of systems.

Reply:
@agraul Could you or someone from the Ion squad have a look at Houssems proposal above.
Thank you

Reply:
Alexander responded in another comment. I'm resolving this.